lib/src/revset.rs

```diff
@@ -773,7 +773,7 @@ static BUILTIN_FUNCTION_MAP: Lazy<HashMap<&'static str, RevsetFunction>> = Lazy:
             RevsetFilterPredicate::DiffContains { text, files },
         ))
     });
-    map.insert("conflict", |function, _context| {
+    map.insert("conflicts", |function, _context| {
```

Comment:
I think it's better to leave the original "conflict" function for a while (with TODO comment about deprecation warning.)

We'll probably need to rename `commit.conflict()` template function for consistency, and it's more likely to break user's stock templates.

I have no idea whether `--conflict`/`--conflicted` should be renamed to `--conflicts`, btw.

Reply:
> I think it's better to leave the original "conflict" function for a while (with TODO comment about deprecation warning.)

Yes, good point (you and @thoughtpolice), especially since we know there are some tools that depend on the CLI (was it lazyjj? jj_tui?).

Reply:
A good sould implemented diagnostics while I wasn't looking :-D
So I've gone ahead to add the deprecation warnings.